scripts/lib/install-node

```diff
@@ -9,6 +9,21 @@ fi
 
 source "$NVM_DIR/nvm.sh"
 node_version=6.6.0
+npm_version=3.10.3
+# if the hashed_value is not changed then exit quickly
+script_hash=`cat scripts/lib/install-node scripts/setup/node-wrapper scripts/setup/npm-wrapper`
+hashed_value=`( echo $node_version ; echo $npm_version ;  echo $script_hash) | sha1sum | awk '{ print $1 }'`
```

Comment:
Instead of three consecutive call to to echo, `echo "${node_version}${npm_version}${script_hash}"` would be more elegant, no?